cvat/apps/engine/tests/test_rest_api.py

```diff
@@ -4562,6 +4612,47 @@ def _send_data_and_fail(*args, **kwargs):
                     image_sizes, StorageMethodChoice.FILE_SYSTEM, StorageChoice.LOCAL,
                     send_data_callback=_send_data_and_fail)
 
+    def _test_api_v2_tasks_id_data_create_can_use_server_rar(self, user):
```

Comment:
I don't think there's any point in testing this in both client and server use cases. The decompression code is the same. Just pick whichever test is easier to write.